### ISSUE TITLE
Clique number check broken for directed graphs

### ISSUE THREAD

**Issue by Willem3141** [graphchecker/moodle-qtype_graphchecker#33]
In GitLab by @Willem3141 on Feb 25, 2020, 19:08

```
basic.py:18: RuntimeWarning: directionality of edges is ignored for directed graphs at cliques.c:908
  actual = getattr(g, method_name)()
```

Meh. We should verify that all checks actually work for directed graphs.

**Comment by Willem3141** [comment]
In GitLab by @Willem3141 on Feb 26, 2020, 11:02

closed via commit e0bfbccebf1a4c4878c8c9c9713aaa2084867935

 *By Willem Sonke on 2020-02-26T10:02:56 (imported from GitLab project)*

**Comment by Willem3141** [comment]
In GitLab by @Willem3141 on Feb 26, 2020, 10:55

Same for the independence number check.

 *By Willem Sonke on 2020-02-26T09:55:41 (imported from GitLab project)*
